app/src/main/java/org/mozilla/fenix/compose/button/TextButton.kt

```diff
@@ -20,23 +24,36 @@ import java.util.Locale
  * Text-only button.
  *
  * @param text The button text to be displayed.
+ * @param shouldUppercaseText Whether or not the text should be uppercase.
+ * @param buttonColors [ButtonColors] for the button.
  * @param onClick Invoked when the user clicks on the button.
  * @param modifier [Modifier] Used to shape and position the underlying [androidx.compose.material.TextButton].
+ * @param contentPaddingValues Padding values for the button's content
  * @param textColor [Color] to apply to the button text.
  */
 @Composable
+@Suppress("LongParameterList")
 fun TextButton(
     text: String,
+    shouldUppercaseText: Boolean = true,
```

Comment:
I don't think `TextButton` should need to know whether or not the text should be uppercase. The caller of `TextButton` should pass the correctly cased text.